### DISCUSSION TITLE
Roadmap

### DISCUSSION THREAD

**Discussion by GrosPoulet** [extesy/hoverzoom#734] [Ideas]
Hello everyone,

@extesy: it would be nice to have a roadmap showing what we are working on & next releases.
Maybe we can use https://trello.com/ ?

Teaser: to be released in June: an image viewer update, with (optional) display of image's details such as dimensions, format, ...

![teaser](https://user-images.githubusercontent.com/23529041/120100559-aaf2b580-c141-11eb-958d-7b3f92bb229e.png)

**Comment by extesy** [comment]
Good idea. It can be done using github's own tools, no need for Trello. I have enabled "projects" tab and created a new empty project so now you can manage https://github.com/extesy/hoverzoom/projects/1?add_cards_query=is%3Aopen. Can you please confirm that you can actually manage it and move issues around?

**Reply by GrosPoulet** [reply]
It works fine :+1: , i added a card to track the image viewer update i talked about.

**Reply by extesy** [reply]
Thanks for confirming. I've added two of the currently most annoying bugs to the TODO column. There are four issues, but three of them actually refer to the same bug (save file shortcut doesn't work on some websites).
